src/openrct2/ride/Station.cpp

```diff
@@ -434,3 +432,19 @@ void ride_set_exit_location(Ride* ride, const int32_t stationIndex, const TileCo
 {
     ride->stations[stationIndex].Exit = location;
 }
+
+int32_t RideStation::GetBaseZ() const
+{
+    return Height * 8;
+}
+
+void RideStation::SetBaseZ(int32_t newZ)
+{
+    Height = newZ / 8;
+}
+
+CoordsXYZ RideStation::GetStart() const
+{
+    TileCoordsXYZ stationTileCoords{ Start.x, Start.y, Height };
+    return stationTileCoords.ToCoordsXYZ();
+}
```

Comment:
I'm thinking if it maybe shouldn't return the `TileCoordsXYZ` and we get to call `ToCoordsXYZ` on the oustide.

Reply:
I haven't done that because we're moving everything to Coords.